versioneer.py

```diff
@@ -1595,6 +1595,7 @@ def run(self):
 
     if "cx_Freeze" in sys.modules:  # cx_freeze enabled?
         from cx_Freeze.dist import build_exe as _build_exe
+
```

Comment:
undo extra newline

Reply:
Then isort will complain... This is not done by me.

Reply:
I removed that newline... And:
```
(py36) E:\pysparkling>isort .
Fixing E:\pysparkling\versioneer.py
Skipped 8 files
```

It's there again...

Reply:
What I could do is run it only against `scripts pysparkling`? Not as nice in my opinion as simple against all python files.
I actually had hoped that the src_paths parameter would take care of this, but seems not.

Reply:
Oh ok. If that one line fixes it, that is fine. Thanks for checking!

I think pylint is already disabled for the entire file. We could do that for isort if there were more changes.

Reply:
Nice, I found the switch! It's as you want now.